docs/users/cli-usage.md

```diff
@@ -29,6 +29,109 @@ Available Commands:
 Use "ddev [command] --help" for more information about a command.
 ```
 
+## Quickstart
+ddev is designed to be as simple as possible to incorporate into existing Wordpress and Drupal workflows. You can start using ddev with any site just by running a few commands.
+
+Below are quickstart instructions for each app type; Wordpress, Drupal 7, and Drupal 8.
+
+**Note:** If you do not have ddev already on your machine, please follow the [installation instructions](https://ddev.readthedocs.io/en/latest/#installation) before beginning the quickstart tutorial. 
+### Wordpress
+To get started using ddev with a Wordpress site, simply clone the site's repository and checkout its directory.
+```
+git clone https://github.com/user/worpress_site
+cd wordpress_site
+```
+From here we can start setting up ddev. Inside of your site's working directory, enter the command:
+```
+ddev config
+```
+
+_Note: ddev config will prompt you for a site name and docroot._
```

Comment:
Just thinking out loud: Maybe they don't know what docroot is? Maybe suggest that it could be current dir (in which case you leave it empty, not entirely intuitive) or "docroot" or "web" or "htdocs" - Maybe ask Rick or Kevin if there's a "most likely" for WP. I think "most likely" for D8 is web.